Commands/Commands/utils.js

```diff
@@ -243,8 +243,8 @@ commands['approve'] = {
                 })
                 return
             } else {
-                if (state[content[0]].approvedUsers.indexOf(message.author.id) > -1) {
-                    message.reply('You\'ve already approved this report.').then(delay(config.timeouts.messageDelete)).then((msg) => {
+                if (state[content[0]].deniedUsers.indexOf(message.author.id) > -1 || state[content[0]].approvedUsers.indexOf(message.author.id) > -1) {
```

Comment:
well, the current way won't work for that then. the report will have both an approval, and a deny in the state. discord testers also takes my approach, and you cannot change your mind.